lib/dartino.dart

```diff
@@ -10,10 +10,9 @@ import 'package:atom/atom.dart';
 import 'package:atom/node/process.dart';
 import 'package:atom/utils/disposable.dart';
 import 'package:atom_dartino/sdk/sdk.dart';
+import 'package:atom_dartino/usb.dart';
```

Comment:
I use relative path references for code in lib/ that also refers to lib/ code. Don't think that's a 1st class convention, but something I do.